backend/routes/user.py

```diff
@@ -39,11 +44,15 @@ async def create_account(user: CreateUserSchema= Body(...)):
 
     database.user.insert_one(user_dict)
 
-    # # generate and save otp for user
+    # generate and save otp for user
     otp = otp_generate_save(user_dict['phone_number'])
-    # # send otp to users phone number
+    # send otp to users phone number
     send_otp_to_phone(user_dict['phone_number'], otp)
 
-    return {'status_code':200, 'message': 'User saved successfully'}
+    return {'status_code': 400, 'message': 'OTP expired. Resend OTP?'}
```

Comment:
Please use this instead 
return {'status_code':200, 'message': 'User saved successfully'}